cdap-support-bundle/src/main/java/io/cdap/cdap/support/handlers/SupportBundleHttpHandler.java

```diff
@@ -42,11 +78,23 @@
 @Path(Constants.Gateway.API_VERSION_3)
 public class SupportBundleHttpHandler extends AbstractAppFabricHttpHandler {
 
+  private static final Logger LOG = LoggerFactory.getLogger(SupportBundleHttpHandler.class);
+  private static final Gson GSON = ApplicationSpecificationAdapter.addTypeAdapters(new GsonBuilder())
+    .registerTypeAdapter(Arguments.class, new ArgumentsCodec())
+    .registerTypeAdapter(ProgramOptions.class, new ProgramOptionsCodec())
+    .registerTypeAdapter(org.apache.twill.internal.Arguments.class, new org.apache.twill.internal.json.ArgumentsCodec())
+    .create();
+  private static final String applicationZip = "application/zip";
+  private static final String contentDescriptionValue = "attachment; filename=\"collect_support_bundle.zip\"";
```

Comment:
nit: Please rename variable to use upper camel case, i.e. CONTENT_DISPOSITION_VALUE. Same for other static final variabes, thanks!